src/test/java/core/basesyntax/MyCalculatorTest.java

```diff
@@ -0,0 +1,257 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class MyCalculatorTest {
+    private static final double DELTA = 0.0001;
+    private static MyCalculator myCalculator;
+
+    @BeforeAll
+    static void beforeAll() {
+        myCalculator = new MyCalculator();
+    }
+
+    @Test
+    void calculate_divisionByZero_notOk() {
+        assertThrows(MyCalculateException.class, () -> myCalculator.calculate(2.0, 0, '/'));
+    }
+
+    @Test
+    void calculate_singIsNotCorrect_notOk() {
+        assertThrows(MyCalculateException.class, () -> myCalculator.calculate(1, 1, '$'));
+    }
+
+    @Test
+    void calculate_additionTwoPositiveValues_Ok() throws Exception {
+        double actual = myCalculator.calculate(7.7, 3.3, '+');
+        double expected = 11;
```

Comment:
```suggestion
        double actual = myCalculator.calculate(7.7, 3.3, '+');
        double expected = 11;
```
You can declare these variables one time as variables of Class.

Reply:
I did it.